### PR TITLE
Issue #2610 Updated minishift architecture diagram

### LINKED ISSUE #2610: Update minishift architecture to reflect new changes
The current minishift architecture seems to be quite old.

minishift_architecture

We have new changes now.
```
...
├── [budhram   4.0K]  iso
│   └── [budhram   4.0K]  centos
│       └── [budhram   4.0K]  v1.12.0
│           └── [budhram   344M]  minishift-centos7.iso
└── [budhram   4.0K]  oc
    └── [budhram   4.0K]  v3.9.0
        └── [budhram   4.0K]  linux
            └── [budhram   208M]  oc
```

### PR DESCRIPTION
**Pull request by budhram**
Fix #2610 

New architecture diagram:
screenshot from 2018-09-24 12-09-47

### CODE REVIEW

File: docs/source/using/minishift-architecture.adoc [hunk 1]
```diff
@@ -2,34 +2,41 @@
...
+ |                      |         |   |   | c1FF                        |        |      |
+ |~/.minishift/         \-=-------|=--|-=-|-=---> OpenShift             |        |      |
+ | |                              |   |   |  |  /-------------------\   |        |      |
+ | +-- cache                      |   |   |  |  |                   |   |        |      |
+ |     |                          |   |   |  |  |                   |   |        |      |
+ |     +-- images                 |   |   |  |  |                   |   |        |      |
+ |     |                          |   |   |  |  | Master   Node     |   |        |      |
+ |     +-- iso                    |   |   |  |  | +-----+  +-----+  |   |        |      |
+ |     |   |                      |   |   |  |  | | {d} |  | {d} |  |   |        |      |
+ |     |   +-- foo-iso         ---|   |   |  |  | |     |  |     |  |   |        |      |
```
Comment (LalatenduMohanty) [inline docs/source/using/minishift-architecture.adoc]: s/foo-iso/iso-version-foo/

File: docs/source/using/minishift-architecture.adoc [hunk 2]
```diff
@@ -2,34 +2,41 @@
...
+ |     |                          |   |   |  |  | |     |  |     |  |   |        |      |
+ |     +-- oc                     |   |   |  |  | |     |  |     |  |   |        |      |
+ |         |                      |   |   |  |  | |     |  |     |  |   |        |      |
+ |         +-- vfoo-version    ---|   |   |  |  | |     |  |     |  |   |        |      |
```
Comment (LalatenduMohanty) [inline docs/source/using/minishift-architecture.adoc]: s/vfoo-versio/oc-version-foo/

**Comment by budhram** [comment]
Updated @LalatenduMohanty

**Comment by LalatenduMohanty** [comment]
The CI failure is not related to the PR

**Pull request merged**